std/experimental/allocator/package.d

```diff
@@ -1612,18 +1612,18 @@ lengths = static array containing the size of each dimension
 Returns:
 An N-dimensional array with individual elements of type T.
 */
-auto makeMultidimensionalArray(uint n, T, Allocator)(auto ref Allocator alloc, size_t[n] lengths)
+auto makeMultidimensionalArray(T, Allocator, size_t N)(auto ref Allocator alloc, size_t[N] lengths...)
```

Comment:
Cool. We've had this chataroo in the previous PR - `N` is not exempt from the usual rules: lowercase for values, uppercase for types. So let's make it lowercase. Thanks!